### ISSUE TITLE
args not added in Run Jest Mode

### ISSUE THREAD

**Issue by ArmandoAA** [firsttris/vscode-jest-runner#39]
Hello,
I'm trying to use  --watch arg in jestrunner.runOptions but this not works, my test ran a single time and stop.

This is config that I use in my VSCode settings.json
 ```
 "jestrunner.runOptions": {
    "args": [
      "--no-cache",
      "--watch",
      "--watch-all"
    ],
    "sourcemaps": "inline",
    "disableOptimisticBPs": true,
  },
```

**Comment by firsttris** [comment]
not 100% sure but:

it could be that those args are only for the debugger? 

and you want to pass commandline arguments to jest?

like in

https://github.com/firsttris/vscode-jest-runner/issues/27

which in not yet supported but highly requested.

**Comment by firsttris** [comment]
seems the feature is really passing the cmd arguments to the jest command, but it was only implemented for the debug mode. 

try "run debug jest"

![image](https://user-images.githubusercontent.com/1709260/65178495-22a9ab00-da59-11e9-9a77-cb7c21422208.png)

**Comment by firsttris** [comment]
i will add this to the normal run jest as well

**Comment by ArmandoAA** [comment]
It works in Debug Mode. Is it possible use debug mode to execute a single file or test?

**Comment by firsttris** [comment]
my understanding is you can do the same with **Debug Jest** as with **Run Jest**. its just additionally in Debug Mode.

**Comment by live-abhishek** [comment]
I want to execute the whole test file with args. My use case is my tests print a whole lot of warning messages on console, which I want to suppress. Passing `--silent` in jest args does that. But in jest runner those args are passed only when I execute `Debug Jest` and not for `Run Jest` or `Run Jest file`. Whats happening is, when I execute the whole file, the amount of warning messages makes is very difficult to find the test failures.

I would really like to pass args even for `Run Jest` and `Run Jest File`.

**Comment by firsttris** [comment]
hey @live-abhishek no one has implement this feature yet, but hopefully we will support this soon for "run Jest" as well.

if your motivated you could contribute it to the extension.

**Comment by firsttris** [comment]
hey @ArmandoAA @live-abhishek 
this is now available with 0.4.4

example settings:

"jestrunner.runOptions": [
          "--coverage",
         "--colors"
]

for debug

"jestrunner.debugOptions": {
   "args": [
     "--no-cache",
     "--watch",
     "--watch-all"
   ],
